src/holon/fixtures/utils/reset_pk_fk_ptr.py

```diff
@@ -26,6 +26,7 @@
         "pk",
         "related_config",
         "related_interactive_element",
+        "rule",
```

Comment:
Should rules be added to fixtures now?

Reply:
Not necessarily, but it could be a relation that exists in the fixture. Therefore it should be incremented. But this script is for manual use anyway so some QC is always there just in case.